lib/install/config/webpacker.yml

```diff
@@ -4,6 +4,7 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_output_path: packs
+  compile: false
```

Comment:
WDYT about flipping this around? Make it `true` by default and `false` in production.